### PR TITLE
Added deferred expressions for wire

### PR DESCRIPTION
**Pull request by pbelange**

### CODE REVIEW

File: tests/.ipynb_checkpoints/test_elements-checkpoint.py [hunk 1]
```diff
@@ -0,0 +1,607 @@
+import numpy as np
```
Comment (giadarol) [inline tests/.ipynb_checkpoints/test_elements-checkpoint.py:1]: I believe this file was added by mistake. --> To be removed

File: xtrack.egg-info/SOURCES.txt [hunk 2]
```diff
@@ -0,0 +1,62 @@
+MANIFEST.in
```
Comment (giadarol) [inline xtrack.egg-info/SOURCES.txt:1]: This file was added by mistake, it should be removed

File: xtrack.egg-info/dependency_links.txt [hunk 3]
```diff
@@ -0,0 +1 @@
+
```
Comment (giadarol) [inline xtrack.egg-info/dependency_links.txt:1]: This file was added by mistake, it should be removed

File: xtrack.egg-info/requires.txt [hunk 4]
```diff
@@ -0,0 +1,5 @@
+numpy>=1.0
```
Comment (giadarol) [inline xtrack.egg-info/requires.txt:1]: This file was added by mistake, it should be removed

File: xtrack.egg-info/top_level.txt [hunk 5]
```diff
@@ -0,0 +1,2 @@
+ducktrack
```
Comment (giadarol) [inline xtrack.egg-info/top_level.txt:1]: This file was added by mistake, it should be removed

File: xtrack/.ipynb_checkpoints/loader_mad-checkpoint.py [hunk 6]
```diff
@@ -0,0 +1,549 @@
+import numpy as np
```
Comment (giadarol) [inline xtrack/.ipynb_checkpoints/loader_mad-checkpoint.py:1]: This file was added by mistake, it should be removed

File: xtrack/beam_elements/.ipynb_checkpoints/elements-checkpoint.py [hunk 7]
```diff
@@ -0,0 +1,845 @@
+from pathlib import Path
```
Comment (giadarol) [inline xtrack/beam_elements/.ipynb_checkpoints/elements-checkpoint.py:1]: This file was added by mistake, it should be removed

File: xtrack/beam_elements/elements_src/.ipynb_checkpoints/wire-checkpoint.h [hunk 8]
```diff
@@ -0,0 +1,55 @@
+#ifndef XTRACK_WIRE_H
```
Comment (giadarol) [inline xtrack/beam_elements/elements_src/.ipynb_checkpoints/wire-checkpoint.h:1]: This file was added by mistake, it should be removed

**Review by giadarol — CHANGES_REQUESTED** [review]

**Comment by pbelange** [comment]
Sorry, getting used to git... I dit a git reset with a force pushed to not commit the extra files... I don't know if I need to open a new merge request?

**Pull request merged**